api/graphql/makeModels.js

```diff
@@ -133,13 +133,14 @@ export default async function makeModels (userId, isAdmin) {
         'start_time',
         'location',
         'announcement',
-        'accept_contributions'
+        'accept_contributions',
+        'is_public'
       ],
       getters: {
         title: p => p.get('name'),
         details: p => p.get('description'),
         detailsText: p => p.getDetailsText(),
-        public: p => (p.get('visibility') === Post.Visibility.PUBLIC_READABLE) || null,
```

Comment:
whoa, what was this?? im intrigued. looks like they were already working on this. did you think about just using what they already had? looks like they were planning for a few different visibility options.

Reply:
Solution - going with the boolean `public` column for now, will investigate Visibility option for the future and we solidify the architecture of public posts & communities.